docs/reference/sections/dom-binding/eq.md

```diff
@@ -0,0 +1,9 @@
+`model.property | eq 'value'`
+
+Returns `true` if the property matches the passed in value.
+
+```html
+  <div rv-show="item.title | eq 'Shirt'">
+    //...
+  </div>
+```
```

Comment:
@gavinballard - General question about structuring formatter reference docs - I wanted to come up with a generic structure for describing how to use the various formatters (this `eq` being the simplest form of that) - do you think this works, or does it need any tweaking before being used across all the formatters?